SPECS/core-packages/core-packages.spec

```diff
@@ -1,90 +1,81 @@
-Name:           core-packages
 Summary:        Metapackage with core sets of packages
+Name:           core-packages
 Version:        0.1
-Release:        21%{?dist}
+Release:        22%{?dist}
 License:        ASL 2.0
-Group:          System Environment/Base
-URL:            http://aka.ms/mariner
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          System Environment/Base
+URL:            https://aka.ms/mariner
 
 %description
 Metapackage holding sets of core packages for different applications.
 
-%package base-image
-Summary:    Metapackage defining the basic set of packages (no kernel) used by images such as VHDs, VHDXs and ISOs.
-
-Requires:   %{name}-container = %{version}-%{release}
-
-%ifarch x86_64
-Requires:   grub2-efi
-Requires:   grub2-pc
```

Comment:
There's quite a few derivatives having a dependency on the `core-packages-base-image` package. Have you verified it won't introduce a regression for them?